src/ui/edit-content/content-attributes/ContentAttributes.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FieldArray,
+} from 'redux-form';
+import { Tabs, Tab } from 'patternfly-react';
+import { FormattedMessage } from 'react-intl';
+
+import AttributeFields from 'ui/edit-content/content-attributes/AttributeFields';
+
+class ContentAttributes extends Component {
+  componentDidMount() {
+    const { onDidMount } = this.props;
+    onDidMount();
+  }
+
+  render() {
+    const { attributes } = this.props;
+
+    return (
+      <Tabs defaultActiveKey={1} animation={false} id="content-attributes-tabs">
+        <Tab eventKey={1} title="English">
+          <FieldArray
+            data-test-id="edit-content-content-attributes-field-array"
+            name="attributes"
+            component={AttributeFields}
+            attributes={attributes}
+          />
+        </Tab>
+        <Tab eventKey={2} title="Italiano">
```

Comment:
I think it should be solved more general way. Right now you have only "English" and "Italian" languages, but what if there are 4 more languages? I think it should be solved this way:
`import { fetchLanguages } from 'state/languages/actions';` inside a container.   Fetch languages inside `mapDispatchToProps` and get those all languages via `import { getActiveLanguages } from 'state/languages/selectors';` inside `mapStateToProps`.  And then, inside `ContentAttributes.js` component map over `languages` and output them too. So, if there is also somewhere any logic only bound to only English and Italian, it must be generalized to N number of languages, (you get language codes inside `languages` array: ['en', 'it', 'ru', 'geo'... etc]

Reply:
You can get `state/languages` code from `app-builder`

Reply:
Thanks. Didn't know that there was already an endpoint for fetching the languages.